sqlglot/dialects/clickhouse.py

```diff
@@ -94,6 +127,10 @@ def _parse_in(
         def _parse_table(
             self, schema: bool = False, alias_tokens: t.Optional[t.Collection[TokenType]] = None
         ) -> t.Optional[exp.Expression]:
+            placeholder = self._parse_placeholder()
```

Comment:
i don't know if this is the right place to put this. perhaps the base parser needs to be adjusted to handle place holders